spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/StreamControllerTests.java

```diff
@@ -572,6 +580,40 @@ public void testDeployWithWildcardProperties() throws Exception {
 	}
 
 	@Test
+	public void testDeployWithCommonApplicationProperties() throws Exception {
+		repository.save(new StreamDefinition("myStream", "time | log"));
+		assertThat(appsProperties.getStream().values(), empty());
+		appsProperties.getStream().put("spring.cloud.stream.fake.binder.host","fakeHost");
```

Comment:
Not a concern if we leave `@EnableConfigurationProperties` in, since the context is marked dirty after each method, I guess?